server/app.js

```diff
@@ -1,6 +1,7 @@
 const express = require("express");
 //const csvToJson = require("csvtojson");
 const { Parser } = require("json2csv");
```

Comment:
You'll probably want to pull the most recent version of dev (with the ES6 changes) and merge it into your branch before completing this PR

Reply:
Yes, git pull --rebase will apply this change for you.

Reply:
I agree. I did this to educate myself, and it worked like Madison described. It will also fix the one git merge conflict for this pull request